src/main/java/core/basesyntax/service/FruitService.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.service;
+
+import core.basesyntax.dao.FruitDao;
+
+public abstract class FruitService {
```

Comment:
you need to follow SOLID principles: 
create 5 separate service for this task:

RiderService - read data from file,
WriterService - write data to file,
ReportMakerService - prepare string report,
TransactionParserService - parse every single row from file to create list of FruitTransaction
FruitShopService - process all list of FruitTransaction using strategy